assets/js/script.js

```diff
@@ -1 +1,9 @@
+var gameCards = document.querySelector("#gameCards");
+gameCards.addEventListener("click", handleClick);
 
+function handleClick (event) {
+  if (event.target.className.indexOf("card-back") === -1) {
+    return;
+  }
+  event.target.className += "hidden";
```

Comment:
If there are already any classes on your element, this will ruin the last class, and hidden will not be applied. Make sure you add a space before your class to prevent this 👍